engine/src/main/java/org/terasology/engine/TerasologyConstants.java

```diff
@@ -77,6 +77,10 @@
      * Path to a module's info file.
      */
     public static final Path MODULE_INFO_FILENAME = Paths.get("module.txt");
+    /**
+     * Name of a saved game's preview image.
```

Comment:
Trivial: You can use one-liners for field javadoc like these, I'd start doing so even in files that use three-liners. So:

`/** Name of a saved game's preview image. /*`

Reply:
I'd prefer to do not change the style and leave it as is. Or I'd need to change all comments for all constants in this class.

Reply:
That's fine :) General practice but not a requirement